src/Microsoft.AspNet.Razor/Parser/TagHelpers/TagHelperBlock.cs

```diff
@@ -42,6 +43,12 @@ public TagHelperBlock(TagHelperBlockBuilder source)
         }
 
         /// <summary>
+        /// Text used as a required prefix when matching HTML start and end tags in the Razor source to available 
+        /// tag helpers.
+        /// </summary>
+        public string Prefix { get; }
```

Comment:
should not have extra members (other than sometimes extra constructor overloads) in product code just for testing.  why test anything related to the prefix?  the important things are matching tags with descriptors and start tags with end tags.